remix-lib/src/execution/execution-context.js

```diff
@@ -141,7 +143,7 @@ function ExecutionContext () {
   }
 
   this.web3 = function () {
-    return this.isVM() ? vms.constantinople.web3vm : web3
+    return this.isVM() ? vms.istanbul.web3vm : web3
```

Comment:
Might be a good time to create a global variable or something so we can update easier next time : 
```typescript
const currentVersion = 'istanbul'
...
return this.isVM() ? vms[currentVersion].web.vm
```